deploy-manage/security/ip-traffic-filtering.md

```diff
@@ -5,19 +5,38 @@ applies_to:
     ece: ga
     eck: ga
     self: ga
-  serverless: unavailable
+  serverless: ga
+navigation_title: "Add IP filters"
 ---
 
-# IP traffic filtering
+# IP filtering
 
-This section covers traffic filtering by IP address or CIDR block.
+This section covers filtering network traffic by IP address or CIDR block.
 
-The way that you configure IP traffic filters depends on your deployment type: 
+The way that you configure IP filters depends on your deployment type.
```

Comment:
```suggestion
The way you configure IP filters, and how filtering is enforced, depends on your deployment type.
```

Reply:
Not sure if that small addon makes sense. The reason is to specify that in ECE, ECH and Serverless this is enforced (or implemented) at platform proxies level, and in self-managed or ECK this is enforced at Elasticsearch cluster level.